nuget/runners/DotnetToolSettings.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<DotNetCliTool>
+  <Commands>
+    <Command Name="nunit3-console" EntryPoint="nunit3-console.dll" Runner="dotnet" />
```

Comment:
This is a chance to simplify the command. Should we use something simpler and easy to type? Maybe just `nunit`? @ChrisMaddock ?

Reply:
Yep - definitely `dotnet nunit` is what we want, if we choose to go ahead with this. (See comment below!)